Code/Framework/AzToolsFramework/AzToolsFramework/Prefab/Spawnable/PrefabToInMemorySpawnableNotificationBus.h

```diff
@@ -0,0 +1,33 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+#include <AzCore/EBus/EBus.h>
+#include <AzFramework/Entity/EntityContext.h>
+
+namespace AzToolsFramework::Prefab
+{
+    //! This EBus is used to send notifications during the conversion of a prefab template to an in-memory spawnable
+    class PrefabToInMemorySpawnableNotifications
+        : public AZ::EBusTraits
+    {
+    public:
```

Comment:
nit, suggest making bus policies explicit for clarity:
```
        static constexpr EBusHandlerPolicy HandlerPolicy = EBusHandlerPolicy::Multiple;
        static constexpr EBusAddressPolicy AddressPolicy = EBusAddressPolicy::Single;
```

Reply:
Done.